index.html

```diff
@@ -0,0 +1,143 @@
+            id="navAction"
+            class="mx-auto lg:mx-0 hover:underline bg-black text-gray-800 font-bold rounded-full mt-4 lg:mt-0 py-4 px-8 shadow opacity-75 focus:outline-none focus:shadow-outline transform transition hover:scale-105 duration-300 ease-in-out"
+          >
+            Action
+          </button> -->
+        </div>
+      </div>
+      <hr class="border-b border-gray-100 opacity-25 my-0 py-0" />
+    </nav>
+
+    <!--Hero-->
+    <div class="pt-24">
+      <div class="container px-3 mx-auto flex flex-wrap flex-col md:flex-row items-center">
+        <!--Left Col-->
+        <div class="flex flex-col w-full md:w-2/5 justify-center items-start text-center md:text-left">
+          <!-- <p class="uppercase tracking-loose w-full">What business are you?</p> -->
+          <h1 class="my-4 text-5xl font-bold leading-tight">
+            A new community for blockchain developer, designers and Dapp Owners
+          </h1>
+          <p class="leading-normal text-xl mb-8">
+            An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀
+          </p>
+          <button style="background: #0086D6" class="mx-auto lg:mx-0 hover:underline text-white rounded-md my-6 py-4 px-8 shadow-lg focus:outline-none focus:shadow-outline transform transition hover:scale-105 duration-300 ease-in-out">
+            Explore now
+          </button>
+        </div>
+        
+        <!--Right Col-->
+        <div class="flex flex-col w-full md:w-3/5 py-6 text-center items-end">
+          <img class="w-full md:w-4/5 z-50 object-right " src="hero.png" />
+        </div>
+      </div>
+    </div>
+
+    <!-- jQuery if we need it
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    -->
+    <script>
+      /*Toggle dropdown list*/
+      /*https://gist.github.com/slavapas/593e8e50cf4cc16ac972afcbad4f70c8*/
+
+      var navMenuDiv = document.getElementById("nav-content");
+      var navMenu = document.getElementById("nav-toggle");
+
+      document.onclick = check;
+      function check(e) {
+        var target = (e && e.target) || (event && event.srcElement);
+
+        //Nav Menu
+        if (!checkParent(target, navMenuDiv)) {
+          // click NOT on the menu
+          if (checkParent(target, navMenu)) {
+            // click on the link
+            if (navMenuDiv.classList.contains("hidden")) {
+              navMenuDiv.classList.remove("hidden");
+            } else {
+              navMenuDiv.classList.add("hidden");
+            }
+          } else {
+            // click both outside link and outside menu, hide menu
+            navMenuDiv.classList.add("hidden");
+          }
+        }
+      }
+      function checkParent(t, elm) {
+        while (t.parentNode) {
+          if (t == elm) {
+            return true;
+          }
+          t = t.parentNode;
+        }
+        return false;
+      }
+    </script>
+  </body>
+</html>
```

Comment:
Missing an endline here.

Reply:
yes, forget it...  added it in the latest commit https://github.com/web3community/web3community.github.io/commit/4caa75e002823833c9e0d1974c1ec52c44dd2515